pkg/security/tests/module_tester.go

```diff
@@ -1089,6 +1131,17 @@ func newTestModule(t testing.TB, macroDefs []*rules.MacroDefinition, ruleDefs []
 		t.Logf("%s entry stats: %s\n", t.Name(), GetEBPFStatusMetrics(testMod.probe))
 	}
 
+	if opts.staticOpts.enableEBPFLess {
+		t.Logf("EBPFLess mode, waiting for a client to connect\n")
+		for i := 0; i < 120; i++ {
```

Comment:
should we use a retry.Do here and return an error if still 0 connection after 120 seconds ?

Reply:
Indeed! Updated on last commit, thanks